app/helpers/mustache_helper.rb

```diff
@@ -174,105 +174,97 @@ def search_page_title
     params[:q].nil? ? '' : [params[:q]].flatten.join(', ')
   end
 
-
-  def navigation_global
-  {
-      :options => {
-        :search_active  => false,
-        :settings_active  => true
-      },
-
-      :logo  => {
-        :url  => "/",
-        :text  => "Europeana Search"
-      },
-
-      :primary_nav  => {
-        :items  => [
-          {
-            :url  => "#",
-            :text  => "Home",
-            :is_current  => true
-          },
-          {
-            :url  => "",
-            :text  => "Channels",
-            :submenu  => {
-              :items  => [
-                {
-                  :url  => "http://google.com",
-                  :text  => "Channel 1"
-                },
-                {
-                  :url  => "http://google.com",
-                  :text  => "Channel 2",
-                  :is_current  => true
-                },
-                {
-                  :url  => "http://google.com",
-                  :text  => "Channel 3"
-                }
-              ]
-            }
-          },
-          {
-            :url   =>  "",
-            :text  =>  "Exhibitions"
-          },
-          {
-            :url   => "",
-            :text  => "Blog"
-          },
-          {
-            :url   => "",
-            :text  => "My Europeana"
-          }
-        ]
-    }
-  }
-  end
-
-  def common_footer
+  def navigation
```

Comment:
Assignment Branch Condition size for navigation is too high. [19/15]<br>Method has too many lines. [92/10]